.github/workflows/lint_client.yml

```diff
@@ -0,0 +1,21 @@
+name: Lint client application
+
+on: [push]  
+
+jobs:
+  client-lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Node 13.12
+      uses: actions/setup-node@v1
+      with:
+        node-version: '13.12'
+    - name: Install dependency
+      run: |
+        cd client
+        yarn install
+    - name: Lint
+      run: |
+        cd client
+        yarn lint
```

Comment:
rly? could we keep EOF newline enforcement?

Reply:
ok, i can add new lines, but there is no way to enforce that in repository configuration/.gitignore files.
In future i will  try to remember to add eof new lines ;)